app/views/transient_registrations/show.html.erb

```diff
@@ -70,6 +70,20 @@
         <% end %>
       </div>
     <% end %>
+
+    <div class="panel wcr-panel wcr-panel-border-all">
+      <h2 class="heading-medium">
+        <%= @transient_registration.company_name %>
+      </h2>
+      <p class="lede">
+        <%= t(".tier.#{@transient_registration.tier.downcase}") %> - <%= show_translation_or_filler(:registration_type) %>
```

Comment:
In the case of lower tier registrations, they never have a `registration_type`.

So if I've understood the code correctly lower tier will always show as **Lower tier - -**. Ideally, for lower tier there wouldn't be any dashes.